### PR TITLE
Move event directories held at Japan to "japan" directory

### PR DESCRIPTION
**Pull request by atoato88**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Trainings in Japan have held 4 times, and it is noisy that folders located at top of this repository. 
This PR moves event directories held at Japan to `japan/` directory.
From now on, we will create directories for upcoming event holding at Japan in `japan/` directory.

**Which issue(s) this PR fixes**:
NONE

**Special notes for your reviewer**:
NONE

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

NONE

### CODE REVIEW

File: japan/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+Hello new contributors!
+
+新コントリビュータの皆さんこんにちわ！
+
+This repository will be used as a safe space for participants in the New Contributor Onboarding Track to familiarize themselves with (some of) the Kubernetes Project's review and pull request processes.
+
+このリポジトリは、(いくつかの)Kubernetsのプロジェクトにおけるレビューやプルリクエストのプロセスに慣れるために、Kubernetesアップストリームトレーニング参加者の安全な場所として使います。
+
+Each subfolder under `japan/` contains assets for kubernetes upstream training holded in event at Japan.
```
Comment (oomichi) [inline japan/README.md:11]: nit: s/event/events/

**Review by oomichi — COMMENTED** [review]
We will have more upstream training events in Japan and this is useful to avoid mess in the root directory.
The change itself is good for me.
just one comment.

/cc @oomichi

File: japan/README.md (hunk 1 above)
Comment (atoato88) [reply japan/README.md]: Thank you to comment. I've fixed this.

**Comment by atoato88** [comment]
/assign @nikhita 
This PR is kind of cleanups, could you review this?

File: japan/README.md (hunk 1 above)
Comment (oomichi) [reply japan/README.md]: Thanks for updating.
The Japanese translation on README.md also seems good for me.

/lgtm

File: japan/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Kubernetes Contributor Playground for Japan
+
+Hello new contributors!
+
+新コントリビュータの皆さんこんにちわ！
```
Comment (shu-mutou) [inline japan/README.md:5]: こんにちは

File: japan/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+This repository will be used as a safe space for participants in the New Contributor Onboarding Track to familiarize themselves with (some of) the Kubernetes Project's review and pull request processes.
+
+このリポジトリは、(いくつかの)Kubernetsのプロジェクトにおけるレビューやプルリクエストのプロセスに慣れるために、Kubernetesアップストリームトレーニング参加者の安全な場所として使います。
+
+Each subfolder under `japan/` contains assets for kubernetes upstream training holded in events at Japan.
+
+各サブフォルダは日本開催のイベントで実施されるKubernetesアップストリームトレーニングのためのアセットを含みます。
+
+Please refers docs in each subfolder for community, discussion, contribution, support, and code of conduct.
```
Comment (shu-mutou) [inline japan/README.md:15]: refers -> refer

File: japan/README.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+Hello new contributors!
+
+新コントリビュータの皆さんこんにちわ！
+
+This repository will be used as a safe space for participants in the New Contributor Onboarding Track to familiarize themselves with (some of) the Kubernetes Project's review and pull request processes.
+
+このリポジトリは、(いくつかの)Kubernetsのプロジェクトにおけるレビューやプルリクエストのプロセスに慣れるために、Kubernetesアップストリームトレーニング参加者の安全な場所として使います。
+
+Each subfolder under `japan/` contains assets for kubernetes upstream training holded in events at Japan.
```
Comment (shu-mutou) [inline japan/README.md:11]: hold -> held

File: japan/README.md (hunk 3 above)
Comment (atoato88) [reply japan/README.md:15]: ok.

File: japan/README.md (hunk 2 above)
Comment (atoato88) [reply japan/README.md:5]: 直します。
I'll fix this!

Comment (oomichi) [reply japan/README.md:5]: こんにちは :-)

File: japan/README.md (hunk 4 above)
Comment (atoato88) [reply japan/README.md]: I've fixed this.

**Comment by guineveresaenger** [comment]
Thank you, friends! Good luck welcoming new contributors from Japan! ❤️  🇯🇵 🙇‍♀️ 

When you're ready, please remove the hold to merge.

/approve
/lgtm
/hold

**Comment by shu-mutou** [comment]
lgtm, Thanks!
/hold cancel

**Comment by atoato88** [comment]
Thank you to approve this!! :tada:

**Pull request merged**